### ISSUE TITLE
Laravel 5.1 issue - Error when creating column

### ISSUE THREAD

**Issue by asmafarhat2006** [cviebrock/eloquent-sluggable#163]
Hi!

Im using laravel 5.1.7.
When i tried to create column in my table via "php artisan sluggable:table tablename" command it gives m an exception "Invalid Argument Exception : There are no commands defined in sluggable namespace" 

Is this package compatible with my used version or am i doing something wrong?
One more detail ill like to mention , is that when i used the vendor publish command to publish config changes it said "Publishing complete for tag[]" and i  didnt see any config file in my folder.

Can you please update me if im doing smthing wrong or missing something

**Comment by muratsplat** [comment]
Are you sure that you did the step?

```
Then, update config/app.php by adding an entry for the service provider.

'providers' => [
    // ...
    'Cviebrock\EloquentSluggable\SluggableServiceProvider',
];
```

**Comment by asmafarhat2006** [comment]
Yes i did.
As for 5.1 i changed the above line to  Cviebrock\EloquentSluggable\SluggableServiceProvider::class

**Comment by muratsplat** [comment]
I also use eloquent-sluggable in  L5.1.8.. It works very well. Now I was tried  to run `php artisan  sluggable:table    test` and works without any exception.

Can you  share  the line of Sluggable in your composer.json . Maybe you use wrong version for L5.1..

**Comment by lazycms** [comment]
when L5.1,run this: php artisan sluggable:table post  slug,console appear  error:
  [Symfony\Component\Debug\Exception\FatalErrorException]
  syntax error, unexpected '{', expecting identifier (T_STRING)

**Comment by cviebrock** [comment]
File or line number reference?

**Comment by 39digits** [comment]
The issue is in the created migration file itself.

SluggableMigrationCreator correctly replaces the {{column}} placeholder in the migration.stub.  The same doesn't happen for {{class}} and {{table}} placeholders leaving those as is in the created migration file (thus the unexpected '{' in the error)

**Comment by cviebrock** [comment]
Fixed in #174 ... thanks!

Can you test the `dev-master` branch to confirm this works now?  And, if you have any ideas on how to write a test for this, that'd be great too!  :)
